src/main/scala/org/constellation/consensus/ConsensusScheduler.scala

```diff
@@ -25,10 +25,9 @@ class ConsensusScheduler(
     .void
 
   val crossTalkConsensus: IO[Unit] = consensusManager.startOwnConsensus().void.handleErrorWith {
-    case error: ConsensusStartError =>
-      IO(logger.debug(error.getMessage))
-    case error: ConsensusError => IO(logger.warn(error.getMessage))
-    case unexpected            => IO(logger.warn(unexpected.getMessage))
+    case error: ConsensusStartError => IO(logger.warn(error.getMessage))
+    case error: ConsensusError      => IO(logger.warn(error.getMessage))
+    case unexpected                 => IO(logger.warn(unexpected.getMessage))
```

Comment:
Is warn the correct level? If it's en error and named as error, shouldn't it be `logger.error`?

Reply:
I'm not sure if situations like:
- No active peers to start consensus
- No tips to start consensus 
- Node has already start own consensus
should be a `warn` or `error`? :thinking: 
What do you think @marcinwadon ?

Reply:
all those above should be `debug` then :) but `unexpected error` should be an `error` anyway

Reply:
fixed